rust/lancedb/src/remote/db.rs

```diff
@@ -88,15 +107,24 @@ impl ConnectionInternal for RemoteDatabase {
             .await
             .unwrap()?;
 
-        let rsp = self
+        let req = self
             .client
             .post(&format!("/v1/table/{}/create/", options.name))
             .body(data_buffer)
             .header(CONTENT_TYPE, ARROW_STREAM_CONTENT_TYPE)
             // This is currently expected by LanceDb cloud but will be removed soon.
-            .header("x-request-id", "na")
-            .send()
-            .await?;
+            .header("x-request-id", "na");
```

Comment:
Not related but I wonder if we should check if this is still needed?

Reply:
I will check. Can address in a follow up if needed.